src/custom/artblocks-engine/index.js

```diff
@@ -46,6 +46,7 @@ export const fetchCollection = async (_chainId, { contract, tokenId }) => {
 export const fetchToken = async (_chainId, { contract, tokenId }) => {
   const url = `https://token.artblocks.io/${contract}/${tokenId}`;
   const { data } = await axios.get(url);
+  const osData = await opensea.fetchTokens(_chainId, [{ contract, tokenId }]);
```

Comment:
you need to have a try catch here and if it fails, fallback to the artblocks image.

Reply:
Also, look at line 33 (for getting collection metadata), we are already getting the opensea metadata, might make sense to also use the opensea image for the collection (not sure if its actually an issue there or not)

Reply:
added the try/catch to both. 

for the collection image here we use the image associated with the tokenId requested - replacing with the OS response returns a different image, so i left that out for now